src/ams.py

```diff
@@ -0,0 +1,187 @@
+"""Module to configure AMS for charms."""
+#
+# Copyright 2024 Canonical Ltd.  All rights reserved.
+#
```

Comment:
Why do we have a different copyright header than in the test files? Lets make it consistent.